wgpu/Cargo.toml

```
@@ -68,7 +72,14 @@ angle = ["wgpu-core?/angle"]
 vulkan-portability = ["wgpu-core?/vulkan-portability"]
 
 ## Enables the GLES backend on WebAssembly only.
-webgl = ["wgpu-core/webgl", "dep:wgpu-hal", "dep:smallvec"]
+webgl = [
+    "wgpu-core/webgl",
```

`wgpu-core/webgl` does enables `wgpu-types/web`, but I am wondering if would be worth being explicit about this:
```suggestion
    "wgpu-core/webgl",
    "wgpu-types/web",
```